.clang-format

```diff
@@ -1,7 +1,5 @@
 BasedOnStyle: Google
-AlignConsecutiveAssignments: true
-AlignConsecutiveDeclarations: true
-AccessModifierOffset: -2
```

Comment:
I do mildly prefer this aligned how we have it but I will allow myself to be overruled if others feel strongly.